src/main/java/pl/ergohestia/ehj1/ivesta/services/DriverService.java

```diff
@@ -14,10 +14,10 @@
 
 public class DriverService extends DriverConfig implements Service<Driver> {
 
-    private static final Logger LOG = LoggerFactory.getLogger(MenuService.class);
+    private static final Logger LOG = LoggerFactory.getLogger(DriverService.class);
```

Comment:
takie zwykłe logi warto nazywać małymi literami. A jeszcze lepiej, skoro i tak macie Lomboka, to użyć adnotacji @Slf4j, która zrobi właśnie to :)